src/server/check-answers/index.js

```diff
@@ -129,16 +129,22 @@ export class SubmitPageController extends QuestionPageController {
   }
 
   generateEmailContent(application) {
-    return Object.values(application.tasks)
-      .flatMap(({ questionPageAnswers }) =>
-        questionPageAnswers
-          .filter(({ page }) => !page.isInterstitial)
-          .map(
-            ({ page, answer }) =>
-              `## ${page.question}\n${answer.emailHtml.replace(/<br \/>/g, '\n')}`
-          )
-      )
-      .join('\n')
+    /**
+     * @type {string[]}
+     */
+    const lines = []
+
+    Object.values(application.tasks).forEach((task) => {
+      lines.push(`# ${task.config.title}`)
+      lines.push('')
+      lines.push('---')
+      task.questionPageAnswers.forEach(({ page, answer }) => {
```

Comment:
We've no longer filtering for interstitial.  I'm trying to figure out whether we did that for a non-type reason in the first place :thinking: